runtime/src/main/java/com/tns/Runtime.java

```diff
@@ -202,17 +202,17 @@ public DynamicConfiguration getDynamicConfig() {
         return dynamicConfig;
     }
 
-    public int getMarkingMode() {
-        String mode = staticConfiguration == null || staticConfiguration.appConfig == null ? "full" : staticConfiguration.appConfig.getMarkingMode();
-        if ("none".equals(mode)) {
-            return 1;
+    @RuntimeCallable
+    public int getMarkingModeOrdinal() {
+        if (staticConfiguration != null && staticConfiguration.appConfig != null) {
+            return staticConfiguration.appConfig.getMarkingMode().ordinal();
+        } else {
+            return MarkingMode.values()[0].ordinal();
```

Comment:
use KnownKeys.MarkingMode.getDefaultValue();

Reply:
I exposed KnownKeys, I think it is fine.